Library/Homebrew/utils/inreplace.rb

```diff
@@ -22,7 +22,10 @@ module Inreplace
     def inreplace(paths, before = nil, after = nil, audit_result = true)
       errors = {}
 
-      Array(paths).each do |path|
+      paths = Array(paths)
+      errors['no files'] = ['no files given to replace'] if paths.empty?
```

Comment:
I didn't come up with better error messages, so any help how to improve it are very welcome